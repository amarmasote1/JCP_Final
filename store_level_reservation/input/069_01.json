[
   {
      "gtin" : "",
      "uom" : "EACH",
      "itemID" : "15345672031",
      "bopus" : "true",
      "lotNumber" : "1534567",
      "shippable" : "true",
      "productLine" : "153",
      "purchaseType" : "Fashion",
      "isSubstitutionAllowed" : "N",
      "itemProcessingTime" : "10"
   },
   {
      "gtin" : "",
      "uom" : "EACH",
      "itemID" : "15345672032",
      "bopus" : "true",
      "lotNumber" : "1534567",
      "shippable" : "true",
      "productLine" : "153",
      "purchaseType" : "Fashion",
      "isSubstitutionAllowed" : "N",
      "itemProcessingTime" : "10"
   },
   {
      "gtin" : "",
      "uom" : "EACH",
      "itemID" : "50012342033",
      "bopus" : "true",
      "lotNumber" : "5001234",
      "shippable" : "true",
      "productLine" : "500",
      "purchaseType" : "Fashion",
      "isSubstitutionAllowed" : "N",
      "itemProcessingTime" : "10"
   },
   {
      "gtin" : "",
      "uom" : "EACH",
      "itemID" : "15345672034",
      "bopus" : "true",
      "lotNumber" : "1534567",
      "shippable" : "true",
      "productLine" : "153",
      "purchaseType" : "Basic6",
      "isSubstitutionAllowed" : "N",
      "itemProcessingTime" : "10"
   }
]            