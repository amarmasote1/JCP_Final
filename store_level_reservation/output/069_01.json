{
    "body": [
        {
            "warnings": [],
            "message": null,
            "nodeType": null,
            "body": null,
            "key": "JCP|15345672031|EACH",
            "status": 0
        },
        {
            "warnings": [],
            "message": null,
            "nodeType": null,
            "body": null,
            "key": "JCP|15345672032|EACH",
            "status": 0
        },
        {
            "warnings": [],
            "message": null,
            "nodeType": null,
            "body": null,
            "key": "JCP|50012342033|EACH",
            "status": 0
        },
        {
            "warnings": [],
            "message": null,
            "nodeType": null,
            "body": null,
            "key": "JCP|15345672034|EACH",
            "status": 0
        }
    ],
    "message": "Successful",
    "status": 221
}