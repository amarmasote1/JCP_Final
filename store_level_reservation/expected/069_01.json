{"body":[{"status":0,"message":null,"key":"JCP|15345672031|EACH","body":null,"warnings":[]},{"status":0,"message":null,"key":"JCP|15345672032|EACH","body":null,"warnings":[]},{"status":0,"message":null,"key":"JCP|50012342033|EACH","body":null,"warnings":[]},{"status":0,"message":null,"key":"JCP|15345672034|EACH","body":null,"warnings":[]}],"status":221,"message":"Successful"}